src/app/components/files/files.component.html

```diff
@@ -55,6 +54,7 @@
                   [format]="column.format"
                   [class]="column.class"
                   [sortable]="column.sortable"
+                  [sortingKey]="column.sortingKey"
```

Comment:
can we do fallback to the `key` and not modify the .json files?
```suggestion
                  [sortingKey]="column.sortingKey || column.key"
```

Reply:
Isn't that value coming from the json file? that is needed as unfortunately the field used in orderBy and the column key sometimes differ (unfortunately), but maybe i'm missing something here :) i'll contact you.

Reply:
It's just optimisation, if the "sortingKey" not found, it will be using "key". and if you want a different sorting you just set the value in the "sortingKey". that can prevent modifying all the columns and adding "sortingKey" that is the same as "key"